common/helpers.go

```diff
@@ -35,9 +35,6 @@ func convertBaseJSHandleTypes(ctx context.Context, execCtx *ExecutionContext, ob
 func convertArgument(
 	ctx context.Context, execCtx *ExecutionContext, arg any,
 ) (*cdpruntime.CallArgument, error) {
-	if gojaVal, ok := arg.(goja.Value); ok {
-		arg = gojaVal.Export()
-	}
 	switch a := arg.(type) {
```

Comment:
Same here, wondering if a temporary assertion to check if an `arg` is of type `goja.Value` and return an error while we stress test the change would be useful.

Reply:
#1182 👍

Reply:
I suggest wrapping this PR up and tackling the assertions in #1182. WDYT?